### PR TITLE
Brazilian Portuguese Merge Request

### PR DESCRIPTION
**Pull request by Kokasgui**
Can you merge this two branches?

### CODE REVIEW

**Comment by maddie480** [comment]
Does not fit on the screen :confused: 

image

File: Dialog/Brazilian Portuguese.txt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+modoptions_collab_speedberrytimerposition= Posição do cronômetro do morango-relógio
+modoptions_collab_speedberrytimerposition_topleft= Canto superior esquerdo
+modoptions_collab_speedberrytimerposition_topcenter= Centro superior
+modoptions_collab_hidespeedberrytimerduringgameplay= Esconder cronômetro do morango-relógio enquanto joga
+
+modoptions_collab_displayendscreenforallmaps= Mostrar tela final para todos os mapas
+modoptions_collab_displayendscreenforallmaps_description= Necessário para verificar IL's no speedrun.com
+
+modoptions_collab_besttimetodisplayinjournal = Melhor tempo a ser mostrado no diário
```
Comment (maddie480) [inline Dialog/Brazilian Portuguese.txt]: You should also remove that space here
```suggestion
modoptions_collab_besttimetodisplayinjournal= Melhor tempo a ser mostrado no diário

```

**Review by maddie480 — CHANGES_REQUESTED** [review]

**Comment by Kokasgui** [comment]
I've already made a new version, I just need the localization team to agree on my changes
If they agree I'll be happy to post the new version in here :)

File: Dialog/Brazilian Portuguese.txt (hunk 1 above)
Comment (Kokasgui) [reply Dialog/Brazilian Portuguese.txt]: I didn't even notice that! I've fixed that now

Comment (Kokasgui) [reply Dialog/Brazilian Portuguese.txt:33]: Sorry, I forgot to send the file
[Brazilian Portuguese.txt](https://github.com/EverestAPI/CelesteCollabUtils2/files/7260224/Brazilian.Portuguese.txt)

**Comment by Kokasgui** [comment]
@max4805 I've updated the version (at least I think I did it right) and confrimed that it now fits the screen
Is it okay to merge the two branches now?

**Review by maddie480 — APPROVED** [review]

**Pull request merged**